src/clickonce/MageCLI/Application.resx

```diff
@@ -270,6 +271,13 @@ Options:
       Specifies if the deployment manifest will include the deployment 
       provider URL.
       
+  -MapFileExtensions &lt;true|false&gt; -ip
```

Comment:
This looks like a copy/paste typo - the switch should be `-mfe`.

Reply:
Ah, yes, you're right, sorry.